recipe/meta.yaml

```
@@ -38,7 +38,7 @@ requirements:
   run:
     - python
     - llvmlite >=0.31.0,<0.33
-    - {{ pin_compatible('numpy', upper_bound='1.18.0') }}
+    - {{ pin_compatible('numpy') }}
```

Shouldn't it be 
```suggestion
    - {{ pin_compatible('numpy', max_pin='1.18') }}
```
?

---

It's being removed based on the consensus formed in PR ( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/51 ).